JourneysSquad/JourneysSquad/View/HistoryOfOperationView.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+
+/// View with a navigation bar, a title and a toolbar item.
+struct HistoryOfOperationView: View {
```

Comment:
I think it's should be Navigation Bar View
```suggestion
struct NavigationBarView: View {
```

Reply:
No, this would be screen History of operation and we create navigation bar on this screen.
You can write or call me and I explain if you don't understand.